client/transactions/list/index.js

```diff
@@ -168,14 +168,20 @@ export const TransactionsList = ( props ) => {
 			<DetailsLink id={ txn.charge_id } parentSegment="transactions" />
 		);
 		const orderUrl = <OrderLink order={ txn.order } />;
+		const orderSubscriptions = txn.order && txn.order.subscriptions;
+		const subscriptionsValue =
+			wcpaySettings.isSubscriptionsActive && orderSubscriptions
+				? orderSubscriptions
+						.map( ( subscription ) => subscription.number )
+						.join( ', ' )
+				: '';
 		const subscriptions =
-			wcpaySettings.isSubscriptionsActive &&
-			txn.order &&
-			txn.order.subscriptions &&
-			txn.order.subscriptions.map( ( subscription, i, all ) => [
-				<OrderLink key={ i } order={ subscription } />,
-				i !== all.length - 1 && ', ',
-			] );
+			wcpaySettings.isSubscriptionsActive && orderSubscriptions
+				? orderSubscriptions.map( ( subscription, i, all ) => [
+						<OrderLink key={ i } order={ subscription } />,
```

Comment:
First, sorry for the drive-by review. Do we have a better choice than to use the index for the `key`? If the order changes [things can get messy behind the scenes.](https://robinpokorny.medium.com/index-as-a-key-is-an-anti-pattern-e0349aece318)

Reply:
No worries at all, thank you for bringing this to our attention! We have this pattern being used in some other parts of the code, though it's not that common, so I created #1789 to track it and fix it in all of the places it's used.